antlr-kotlin-examples-mpp/gradle

```diff
@@ -0,0 +1 @@
+../gradle
```

Comment:
do we need also the `gradle` command? Normally I just use `gradlew` and `gradle` is the one installed on the machine globally

Reply:
Hi @tfomassetti,
this is just a symlink